pytest_postgresql/config.py

```diff
@@ -28,17 +28,18 @@ class PostgresqlConfigDict(TypedDict):
 def get_config(request: FixtureRequest) -> PostgresqlConfigDict:
     """Return a dictionary with config options."""
 
-    def get_postgresql_option(option: str) -> Any:
+    def get_postgresql_option(option: str, default: Any = None) -> Any:
         name = "postgresql_" + option
-        return request.config.getoption(name) or request.config.getini(name)
+        return request.config.getoption(name) or request.config.getini(name) or default
 
     load_paths = detect_paths(get_postgresql_option("load"))
 
     return PostgresqlConfigDict(
         exec=get_postgresql_option("exec"),
         host=get_postgresql_option("host"),
         port=get_postgresql_option("port"),
-        port_search_count=get_postgresql_option("port_search_count"),
+        # Parse as int, because if it's defined in an INI file then it'll always be a string
+        port_search_count=int(get_postgresql_option("port_search_count", default=5)),
```

Comment:
I'd defer to keeping all defaults in single place either defaults for ini or here

Reply:
Sounds good! I'll keep them in INI. This should also fix the bug where my `return request.config.getoption(name) or request.config.getini(name) or default` code was returning None for any falsey config setting

Reply:
(I could change it to be more careful about only coalescing when request.config.getoption(name) is None)